test/logger-usage/src/main/java/org/elasticsearch/test/loggerusage/ESLoggerUsageChecker.java

```diff
@@ -72,10 +72,10 @@
     public static void main(String... args) throws Exception {
         System.out.println("checking for wrong usages of ESLogger...");
         boolean[] wrongUsageFound = new boolean[1];
-        checkLoggerUsage(wrongLoggerUsage -> {
-            System.err.println(wrongLoggerUsage.getErrorLines());
-            wrongUsageFound[0] = true;
-        }, args);
+//        checkLoggerUsage(wrongLoggerUsage -> {
```

Comment:
I still need to figure out how to fix this test

Reply:
This test is still commented out. We should still make it work?

Reply:
absolutely, thanks for reminding!